File: AbsManagement_BackEnd/AbsManagementAPI/Core/Models/CanBo/CanBoModel.cs
using AbsManagementAPI.Core.Entities;

namespace AbsManagementAPI.Core.Models.CanBo
{
    public class CanBoModel
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string HoTen { get; set; }
        public string SoDienThoai { get; set; }
        public DateTimeOffset NgaySinh { get; set; }
        public string Role { get; set; }
        public List<string> NoiCongTac { get; set; }
    }
}

File: AbsManagement_FrontEnd/fe-admin/src/apis/canBo/canBoModel.ts
interface CanBo {
    email:string;
    hoTen:string;
    soDienThoai: string;
    role:string;
    noiCongTac: string[];
}

export interface CanBoModel extends CanBo{
    id: number;
    ngaySinh: Date;
}

export interface CapNhatCanBoModel extends CanBo{
    id: number;
    ngaySinh: any;
    phuong:string,
    quan:string;
}

export interface ThemMoiCanBoModel extends CanBo{
    ngaySinh: any;
    phuong:string,
    quan:string;
}
